opentelemetry-distro/setup.cfg

```diff
@@ -0,0 +1,56 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[metadata]
+name = opentelemetry-distro
+description = OpenTelemetry Python Distro
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = OpenTelemetry Authors
+author_email = cncf-opentelemetry-contributors@lists.cncf.io
+url = https://github.com/open-telemetry/opentelemetry-python/tree/master/opentelemetry-distro
+platforms = any
+license = Apache-2.0
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+python_requires = >=3.5
+package_dir=
+    =src
+packages=find_namespace:
+zip_safe = False
+include_package_data = True
+install_requires =
+    opentelemetry-api == 0.17.dev0
+    opentelemetry-sdk == 0.17.dev0
+    opentelemetry-exporter-otlp == 0.17.dev0
```

Comment:
Is this dependency needed?

Reply:
It is needed if we expect the default OTLP configuration to work

Reply:
Is it needed explicitly by this package though? I can see how to get the entire auto-instrumentation working you would need to install `opentelemetry-exporter-otlp` but shouldn't that be handled in documentation?

Reply:
Since this is mostly an example, I think defaulting to OTLP is fine but could be an extra dep so package can be installed as `opentelemetry-distro[otlp]` or `opentelemetry-distro[jaeger]` etc. Another option is to rely on the bootstrap command to install the desired exporter.